Plugins/MongoDataStore/server/index.js

```diff
@@ -0,0 +1,47 @@
+// Overall PenPal coordinating server code
+import PenPal from "meteor/penpal";
+import _ from "lodash";
+import { Mongo } from "meteor/mongo";
+
+// Plugin-specific info
+import Plugin from "./plugin.js";
+import Manifest from "./manifest.json";
+
+// Register the plugin
+PenPal.registerPlugin(Manifest, Plugin);
+PenPal.MongoCollections = {}
+
+PenPal.DataStore.update = (plugin_name, table, selector, update) => {
+    return PenPal.MongoCollections[`${plugin_name}${table}`].update(selector, update)
```

Comment:
just as a point of readability, does `BaseDatastore1` look better or `Base.Datastore1` (or some other delimeter)? Do we care?

Reply:
let's just leave it for now